odoo_office365_contacts==UPDATED/odoo_microsoft_account/models/oauth_provider.py

```
@@ -0,0 +1,73 @@
+        refresh_token=None,
+        context=None,
+    ):
+        data = dict(
+            grant_type=type_grant,
+            redirect_uri=request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("web.base.url")
+            + "/auth_oauth/microsoft/signin",
+            client_id=oauth_provider_rec.client_id,
+            client_secret=oauth_provider_rec.secret_key,
+        )
```

Pylint (3.3.7)

[refactor] Consider using '{"grant_type": type_grant, "redirect_uri": request.env['ir.config_parameter'].sudo().get_param('web.base.url') + '/auth_oauth/microsoft/signin', ... }' instead of a call to 'dict'.

(R1735)